### DISCUSSION TITLE
Search for people does not include all fields in Sharepoint profile

### DISCUSSION THREAD

**Discussion by frankasphaug** [microsoft-search/pnp-modern-search#2335] [Q&A] [answered]
**Version used**
1.0.21.0

**Describe the bug**
We use the search vertical and have a tab for people.
The "Result Source Identifier" column has this value b09a7990-05ea-4af9-81ef-edfab16c4e31. (The default setting)

When we search we don't get hits on fields we have added extra in the Sharepoint Profile.

**To Reproduce**
Add a new field on the Sharepoint profile and add data to the field for some users. Test search

**Expected behavior**
Would expect that we would find users with hits on new fields in Sharepoint profile

**Screenshots**
N/A

**Desktop (please complete the following information):**
Edge

**Comment by wobba** [comment, marked as answer]
Seems API related or mis-configuration. Could be https://www.techmikael.com/2014/11/how-to-make-sure-user-profile.html

**Reply by frankasphaug** [reply]
Thanks
It works now!

**Comment by frankasphaug** [comment]
Looks like a winner but where do I find the managed property settings page which includes the Advanced Searchable Settings button?

**Reply by wobba** [reply]
Go to https://tenant-admin.sharepoint.com/_layouts/15/searchadmin/ta_listmanagedproperties.aspx?level=tenant and click the blue button
![image](https://user-images.githubusercontent.com/51104/188631213-a43a4ecb-93bb-479a-8bc1-304649974a6a.png)
